src/content/docs/logs/logs-context/configure-logs-context-ruby.mdx

```diff
@@ -15,11 +15,150 @@ redirects:
+
+<CollapserGroup>
+
+  <Collapser
+    id="1-agent"
+    title="Option 1 (simplest): Let the agent decorate and forward your logs."
+  >
+
+  This is the simplest approach, and it's a great choice for developers who may not have the access or interest in setting up a log forwarder, or for accounts that want to see the power of logs and other linking metadata in context of their apps, without a lot of overhead.
+  
+  Using this option, your logs will include `span.id`, `trace.id`, `hostname`, `entity.guid`, and `entity.name`. These attributes automatically link your logs to spans, traces, and other telemetry, making it easier to troubleshoot.
+
+  All you need to do is install an agent version with log forwarding capabilities ([Ruby agent 8.6.0 or higher](https://docs.newrelic.com/docs/release-notes/agent-release-notes/ruby-release-notes/)), and update your configuration to enable forwarding.
+
+  newrelic.yml:
+  ```
+  application_logging:
+    forwarding:
+      enabled: true
+  ```
+
+  Environment variable:
+  ```
+  NEW_RELIC_APPLICATION_LOGGING_FORWARDING_ENABLED=true
+  ```
+
+  This option may be on by default in a future agent version.
+
+  **Optional adjustments:**
+
+  Once this is enabled, you also have control over the maximum number of logs sent to New Relic every minute. The default value is 10,000. If more than 10,000 logs are received in a 60-second window, your logs will begin to be sampled.
+  
+  Set it to a higher number to receive more logs. Set it to a lower number to receive fewer logs. Any integer is valid.
+
+  newrelic.yml:
+  ```
+  application_logging:
+    forwarding:
+      max_samples_stored: 10_000
+  ```
+
+  Environment variable:
+  ```
+  NEW_RELIC_APPLICATION_LOGGING_FORWARDING_MAX_SAMPLES_STORED: 10000
+  ```
+
+  If you have an existing log forwarding solution and are updating your agent to use automatic logs in context, be sure to **disable your old log forwarder**. Otherwise, your app will be sending double log lines. Depending on your account, this could result in double billing. For more information, follow the procedures to disable your [specific log forwarder](/docs/logs/forward-logs/enable-log-management-new-relic#log-forwarding).
+  </Collapser>
+
+  <Collapser
+    id="2-decorate"
+    title="Option 2: Let the agent decorate your logs."
+  >
+
+  Already have a log forwarder you like? We've got you covered! Language agents can decorate your logs with the linking metadata needed to provide access to automatic logs-in-context features.
+
+  1. If you are currently sending logs using the old logs-in-context method with the `NewRelic::Agent::Logging::DecoratingFormatter`, be sure to remove that code from your [specific log forwarder](/docs/logs/forward-logs/enable-log-management-new-relic#log-forwarding) before enabling local decoration. Otherwise, your app may send double log lines. Depending on your account, this could result in double billing.
```

Comment:
@barbnewrelic  - After chatting with Tanna, we decided to rephrase this sentence to be a bit more clear:

```suggestion
  1. This option should not be used with in-agent forwarding. Using an external log forwarder to send logs to New Relic while in-agent forwarding is enabled will cause your logs to be sent up twice to New Relic. Depending on your account, this may result in double billing. 
```